wesnoth-did.po

```diff
@@ -1607,12 +1654,15 @@ msgid ""
 "after the corpses are all rotted to nothing but bones, the bridge between "
 "the land of death and life is weakened.” If I use that knowledge..."
 msgstr ""
+"Jak to vlastně Darken Volk říkal? „Po velkém krveprolití, dokonce i když "
+"jsou všechny mrtvoly vyhnilé na kost, je most mezi zemí smrti a zemí života "
+"oslaben.“ Pokud tuto znalost použiji..."
 
 #. [message]: speaker=Malin Keshar
 #: data/campaigns/Descent_Into_Darkness/scenarios/03_A_Haunting_in_Winter.cfg:1826
 msgid ""
 "... I should be able to sense the soul past the barrier of life and death..."
-msgstr ""
+msgstr "... Měl bych být schopen provést duši přes bariéru života a smrti..."
```

Comment:
vidím tam něco trochu jiného
```suggestion
msgstr "... Měl bych být schopen vycítit duši za bariérou života a smrti..."
```

Reply:
Vím, že to není úplně dobře. Bez kontextu bych to přeložil přesně takhle. Ale v tom monologu následuje to, že Malin dokázal vyvolat ducha. Takže mi přišlo logičtější, že dokáže s duší něco udělat.
Asi by byla vhodnější změna přímo v originálu...

Reply:
Rozumím, nechci vás učit překládat.

Je to váš boj, když se chcete sám nořit do temnoty logického vysvětlování vyvolávání duchů.

Nevím, jak se to hraje teď, ale řekl bych, že hra je v těch místech sama akční (duch je vyvolán, Malin dostane přes prsty...), takže já osobně bych se nesnažil to ještě dokonale vysvětlit, spíš to napsat tak, aby to  neskřípalo ve hře.

Reply:
Měl bych být schopen vycítit duši bloudící kolem bariéry života a smrti?

Reply:
na hranici mezi životem a smrtí (používá se více)

na rozhraní života a smrti (používá se méně)

Reply:
Je to tam.